docs/docs.Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM frolvlad/alpine-glibc:alpine-3.9 AS builder
```

Comment:
Could this be an official base image? Maybe an official Rust image?

Reply:
We just need to the correct glibc library, the version we need isn't precompile for Alpine Linux, hence the use of a third party image. I have checked the dockerfile and this images is used by the `Deno` project. We can use an official Rust image, but it is much larger. I will switch to the official rust image, but if image pull starts slowing our build time we may want to switch back.